src/Host/Client/Impl/Host/RHost.cs

```diff
@@ -401,6 +401,15 @@ public sealed partial class RHost : IDisposable, IRExpressionEvaluator {
                                 _callbacks.DirectoryChanged();
                                 break;
 
+                            case "library":
+                                _callbacks.ViewLibrary();
```

Comment:
Also, I'm not sure we really need it (yet, anyway). A more reliable way to override `setwd` would be to do the same thing we do for `parse` - i.e. override the C implementation in `R_FunTab` (the place where `.Internal` looks things up).

Although for this particular thing, I wouldn't even bother overriding; instead, make the host code check for current directory every time it hits a new prompt, compare it with the old value, and if it's different, send `setwd`. It's common enough and low level enough that I'm not concerned with it being part of core host functionality.